code/game/objects/items/food/bread.dm

```diff
@@ -256,6 +256,55 @@
 	tastes = list("bread" = 1)
 	foodtypes = GRAIN
 	venue_value = FOOD_PRICE_CHEAP
+	/// whether this is in fake swordplay mode or not
+	var/fake_swordplay = FALSE
+
+/obj/item/food/baguette/examine(mob/user)
+	var/examine_list = ..()
+	if(user.mind?.miming)
+		examine_list += span_notice("You can wield this like a sword by right clicking it.")
+
+/obj/item/food/baguette/attack_self_secondary(mob/user, modifiers)
+	. = ..()
+	if(!user.mind?.miming)
+		return
+	fake_swordplay ? end_swordplay(user) : begin_swordplay(user)
```

Comment:
just use an if statement, ternaries should be used as actual expressions rather than statements